common-lib/src/main/scala/com/gu/mediaservice/model/leases/LeaseByMedia.scala

```diff
@@ -0,0 +1,42 @@
+package com.gu.mediaservice.model
+
+import play.api.libs.json._
+import play.api.libs.functional.syntax._
+import org.joda.time.DateTime
+
+
+case class LeaseByMedia(
+  leases: List[MediaLease],
+  lastModified: Option[DateTime],
+  length: Int,
+  current: Option[MediaLease]
+)
+case object LeaseByMedia {
+  implicit val dateTimeFormat = DateFormat
+  implicit val writer = new Writes[LeaseByMedia] {
+    def writes(leaseByMedia: LeaseByMedia) = JsObject(Seq(
+      "leases" -> Json.toJson(leaseByMedia.leases),
+      "lastModified" -> Json.toJson(leaseByMedia.lastModified),
+      "length" -> Json.toJson(leaseByMedia.length),
+      "current" -> Json.toJson(leaseByMedia.current)
+    ))
+  }
+
+  def apply(leases: List[MediaLease]): LeaseByMedia = {
+    def sortLease(a: MediaLease, b: MediaLease) =
+      a.createdAt.isAfter(b.createdAt)
+
+    val sortedLeases = leases
+      .sortWith(sortLease)
+
+    val currentLease = sortedLeases
+      .filter(_.active)
+      .headOption
+
+    val lastModified = sortedLeases
+      .headOption
+      .map(_.createdAt)
+
+    LeaseByMedia(sortedLeases, lastModified, leases.length, currentLease)
```

Comment:
I'm not sure it matters, but I think this might be better named "LeaseCollection" or "LeaseList" or something. Theres's nothing in here that's Media specific. Just a thought, in case we might be able reuse it at somepoint